### ISSUE TITLE
Use Wazuh Indexer and Wazuh Dashboard Docker images

### ISSUE THREAD

**Issue by AlexRuiz7** [wazuh/wazuh-indexer#260]
## Description

Related issue: #250

Our Docker Compose projects for the 3rd party integrations currently use "fake" Wazuh Indexer and Wazuh Dashboard Docker images, as they actually use OpenSearch's images.

As we have our own Docker images, we would like to replace these with ours, either using the Docker images generated in #192 or https://github.com/wazuh/wazuh-docker. The images mentioned first are the preferred ones as we generate them and have full control over them, although these are not the official Docker images. In that case, we would need to upload the images to a repository (quay.io).

## Tasks

- [ ] Replace OpenSearch's Docker containers with ours on the Docker Compose projects inside the integrations/ folder.

**Comment by QU3B1M** [comment]
In an initial analysis of this issue, I've found some concerns that we must discuss before applying the images to our Docker compositions.
1. Who and how should build and publish our images?
2. Which images are the officials? the ones on this repo or the ones from [wazuh-docker](https://github.com/wazuh/wazuh-docker)?

In my opinion, taking in account the approach of  "each project build its own packages" it is correct for the _wazuh-indexer_ Docker images to be built in its own repository being easier to maintain it.

I've made a solution proposal in the PR #476 that is for now incomplete, but the idea is to have a bash script that handles the build and publish process that can be later consumed in a GHA Workflow
